cog/daily_charge.py

```
@@ -23,95 +23,257 @@
-        self.embed.set_thumbnail(url="https://http.cat/images/404.jpg")
-        self.embed.add_field(name="這裡似乎沒有打雷…", value="  ⛱️", inline=False)
-        self.embed.add_field(name="到「每日充電」頻道試試吧！", value="", inline=False)
-        # 其他文案：這裡似乎離無線充電座太遠了，到「每日充電」頻道試試吧！ 待商議
-        await interaction.response.send_message(embed=self.embed, ephemeral=True)
+    # TODO: inherit a MySQLCursorAbstract to add method about these or consider to add self.cursor
+    def reward(
+        self,
+        user: discord.User | discord.Member,
+        last_charge: datetime,
+        executed_at: datetime,
+        cursor,
```

Ruff (0.8.2)

Missing type annotation for function argument `cursor`

(ANN001)